scripts/fetchODCConfig.json5

```diff
@@ -1,6 +1,8 @@
+// Contains all configs used by scripts/fetchODCDataFiles.js for updating newly published ODC data files of
+// COVID-19 assessment centre locations into WeCount Github repository.
 {
 	// The webpage URL where the data file for COVID-19 assessment centre locations is published.
```

Comment:
This comment should be changed to indicate we expect the URL to report a JSON feed referring to some CSV, etc.